include/grpcpp/impl/codegen/server_callback_impl.h

```diff
@@ -506,30 +503,30 @@ class ServerReadReactor : public internal::ServerReactor {
   // customization point.
   virtual void InternalBindReader(ServerCallbackReader<Request>* reader) {
     grpc::internal::ReleasableMutexLock l(&reader_mu_);
```

Comment:
does this have to be releasable now?

Reply:
So.... I kept this Releasable so that I don't have to construct an empty PreBindOperations and then set it up by move assignment since Status is not constexpr-constructible. We should probably change that part and make OK status constexpr-constructible (e.g., only have the strings in place when they are used)

Reply:
So, yeah, I kept the TODO in only one place but meant it for all of them. I should make that clear.

Reply:
Oh I see. That makes sense now.